mockData/dbHelper.js

```diff
@@ -1,8 +1,19 @@
 const Restaurant = require('./data.js');
 const db = require('./index.js')
+const mongoose = require('mongoose');
 
 var getRestaurantByID = function(search, callback) {
   var query;
+
+  //req.params returns string. Convert search to number if it is a number
+  var memSearch = search;
+  if(Number(search) !== NaN) {
+    search = Number(search);
+  } else {
```

Comment:
Is this else statement necessary? If the if-statement doesn't activate, aren't search and memSearch already equal?